src/main/java/it/gov/pagopa/afm/calculator/repository/CosmosRepository.java

```diff
@@ -36,8 +36,8 @@ public class CosmosRepository {
 
   @Autowired UtilityComponent utilityComponent;
 
-  @Value("${pspPoste.id}")
-  private String pspPosteId;
+  @Value("#{'${pspPoste.id}'.split(',')}")
+  private List<String> pspPosteId;
```

Comment:
why use a list?

Reply:
Criteria method NOT_IN requests a list, plus could be useful if other psp are added in future